source/sass/print.scss

```diff
@@ -0,0 +1,58 @@
+@media print {
+  @page {
+    margin: 2cm;
+  }
+
+  body {
+    color: $black;
+    background: $white;
+    width: 100%;
+  }
+
+  video {
+    display: none;
+  }
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
+  ul,
+  ol,
+  img {
```

Comment:
We probably want to take `ul` and `ol` out here - page breaks after a list should be fine?

Reply:
`ul` and `ol` removed. Now I wonder if `img` should be removed from here as well?

@sabrinang the rule `page-break-after: avoid;` is to make sure page break is avoided after the elements we specificed here: `h1` to `h6` and image.